db/schema.rb

```diff
@@ -11,7 +11,56 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20150701042203) do
+ActiveRecord::Schema.define(:version => 20160126231512) do
+
+  create_table "credence_answers", :force => true do |t|
+    t.integer  "credence_question_id"
```

Comment:
This seems to be a foreign key referring to the table credence_questions. It's useful to index all foreign keys so you can do credence_question.credence_answers without having to scan through all records in the credence_answers table. This is usually not a big problem when the amount of SQL records is small, but as it grows it most likely will.

There are different schools on adding indexes. Some say "add indexes to foreign keys by default" while others say, "add no indexes and wait until performance issues start showing".

Please make your own choice for this scenario, and we'll go with that :)